src/components/shared/NavBar.tsx

```diff
@@ -53,56 +51,85 @@ export const NavBar = () => {
-          </Link>
-        </div>
-        <div className={getTabClass('/About')}>
-          <Link to={withLocaleUrl("About")}>
-            {isMobileDeviceOrTablet ?
-              <FontAwesomeIcon icon={faUsers} size="lg" /> :
-              Translate('About')}
-          </Link>
-        </div>
-        <div className={getTabClass('/Partnerships')}>
-          <PartnershipsDropDown />
-        </div>
-        <div className={getTabClass('/Language') + " cursor"} onClick={() => changeLanguages()}>
-          <div>
-            {state.language === LanguageType.english ?
-              (isMobileDeviceOrTablet ? "Fr" : "Français") :
-              (isMobileDeviceOrTablet ? "En" : "English")}
-          </div>
-        </div>
-      </div>
+      { isMobileDeviceOrTablet ? mobileNav() : desktopNav(isMobileDeviceOrTablet,getTabClass, changeLanguages, state.language) }
     </header >
   )
+}
+
+const mobileNav = () => {
+  return (<Dropdown icon='bars' direction='left'>
+  <Dropdown.Menu>
+    <Dropdown.Item>
+      <Link to={withLocaleUrl("Explore")}>
+          {Translate('Explore')}
+      </Link>
+    </Dropdown.Item>
+    <Dropdown.Item>
+      <Link to={withLocaleUrl("Analyze")}>
+        {Translate('Analyze')}
+      </Link>
+    </Dropdown.Item>
+    <Dropdown.Item>
+      <Link to={withLocaleUrl("Data")}>
+        {Translate('Data')}
+      </Link>
+    </Dropdown.Item>
+    <Dropdown.Item>
+      <Link to={withLocaleUrl("Publications")}>
+        {Translate('Publications')}
+      </Link>
+    </Dropdown.Item>
+    <Dropdown.Item>
+      <Link to={withLocaleUrl("About")}>
+        {Translate('About')}
+      </Link>
+    </Dropdown.Item>
+    <Dropdown.Item>
+      <PartnershipsDropDown />
+    </Dropdown.Item>
+  </Dropdown.Menu>
+</Dropdown>)
+}
+
+const desktopNav = (isMobileDeviceOrTablet: boolean, getTabClass: any, changeLanguages:any, language: any) => {
+  return (<div className={`App-tabs col-auto space-evenly p-0 ${isMobileDeviceOrTablet ? 'fill' : ''}`}>
+  <div className={getTabClass('/Explore')}>
+    <Link to={withLocaleUrl("Explore")}>
+      {Translate('Explore')}
+    </Link>
+  </div>
+  <div className={getTabClass('/Analyze')}>
+    <Link to={withLocaleUrl("Analyze")}>
+      {Translate('Analyze')}
+    </Link>
+  </div>
+  <div className={getTabClass('/Data')}>
+    <Link to={withLocaleUrl("Data")}>
+      {Translate('Data')}
+    </Link>
+  </div>
+  <div className={getTabClass('/Publications')}>
+    <Link to={withLocaleUrl("Publications")}>
+      {Translate('Publications')}
+    </Link>
+  </div>
+  <div className={getTabClass('/About')}>
+    <Link to={withLocaleUrl("About")}>
+      {Translate('About')}
+    </Link>
+  </div>
+  <div className={getTabClass('/Partnerships')}>
+    <PartnershipsDropDown />
+  </div>
+  <div className={getTabClass('/Language') + " cursor"} onClick={() => changeLanguages()}>
+    <div>
+      {language === LanguageType.english ?
+        (isMobileDeviceOrTablet ? "Fr" : "Français") :
```

Comment:
probably don't need this ternary here because we're certain this will only be rendered on desktop right